### PR TITLE
Ews Mail Sender -Add command reply-mail

### PR DESCRIPTION
**Pull request by Bargenish**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/29245

## Description
- Added a new command named `reply-mail` in order to reply to a given message instead of sending a new one.
- Modified tpb and it's instance config
- Updated Docker image to use exchangelib 1.12.5 in order to add support for ReplyItem object.
- Updated README with new command and converted it to md

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by Bargenish** [comment]
@yaakovi code is ready to review . need to change the docker image after https://github.com/demisto/dockerfiles/pull/3044 will get merged

File: Packs/EWSMailSender/Integrations/EWSMailSender/EWSMailSender.py [hunk 1]
```diff
@@ -273,7 +324,7 @@ def test_module():
     global IS_TEST_MODULE
     IS_TEST_MODULE = True
     BaseProtocol.TIMEOUT = 20
-    get_account(ACCOUNT_EMAIL)
+    get_account(ACCOUNT_EMAIL).root
```
[the remark this answers is not in the document]
Comment (Bargenish) [reply Packs/EWSMailSender/Integrations/EWSMailSender/EWSMailSender.py:327]: yes. following our meeting with Ika and Anar, this test is more comprehensive and should fail for the account that doesn't work.

File: Packs/EWSMailSender/Integrations/EWSMailSender/EWSMailSender.py [hunk 2]
```diff
@@ -135,6 +137,24 @@ def send_email_to_mailbox(account, to, subject, body, bcc=None, cc=None, reply_t
...
+                                attachments=[]):
+    item_to_reply_to = account.inbox.get(id=inReplyTo)
+    if isinstance(item_to_reply_to, ErrorItemNotFound):
+        raise Exception(item_to_reply_to)
+
+    subject = subject or item_to_reply_to.subject
+    message_body = HTMLBody(html_body) if html_body else body
+    m = item_to_reply_to.create_reply(subject='Re: ' + subject, body=message_body, to_recipients=to, cc_recipients=cc,
+                                      bcc_recipients=bcc)
+    m = m.save(account.drafts)
```
[the remark this answers is not in the document]
Comment (Bargenish) [reply Packs/EWSMailSender/Integrations/EWSMailSender/EWSMailSender.py:150]: no, the itemReply object doesnt have a `send_and_save` attribute.

**Review by yaakovi — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@yaakovi Doc review completed.

**Pull request merged**